docs/reference/commands/disk.rst

```diff
@@ -48,7 +62,7 @@ Flags:
    --db-device string    The device used for the DB
    --db-encrypt          Encrypt the DB device prior to use
    --db-wipe             Wipe the DB device prior to use
-   --encrypt             Encrypt the disk prior to use
+   --encrypt             Encrypt the disk prior to use (only block devices)
```

Comment:
i would s/only block devices/loopback devices not supported/

a loopback device is technically a block device